remote/remote_state_gcs.go

```diff
@@ -14,7 +14,6 @@ import (
 	"github.com/gruntwork-io/terragrunt/options"
 	"github.com/gruntwork-io/terragrunt/shell"
 	"github.com/gruntwork-io/terragrunt/util"
-	"github.com/hashicorp/terraform/helper/pathorcontents"
```

Comment:
This package is no longer available, so we need an alternate function which I created down below.